public/js/chat.js

```diff
@@ -14,6 +14,13 @@
   let cancel = document.querySelector('#cancel');
   socket.emit('privateRoom', { "room": "private room" });
 
+
+  message.addEventListener("keyup", event => {
+    if(event.key !== "Enter") return;
+    sendbtn.click();
+    event.preventDefault();
```

Comment:
move this before line number 19. at the start of callback

Reply:
didn't get what you are trying to say